### PR TITLE
Add build instructions for x86_64 Linux platform

### LINKED ISSUE #595: Add instructions to README on how to run node with testnet and mainnet
As was fairly [pointed out on Reddit](https://www.reddit.com/r/cardano/comments/f879vw/docker_image_for_cardanonode_160_is_here/): there no instructions on
> ... how to connect the current code-base to either Byron mainnet or testnet, do you have some docs on that or is it going to be more of a trial and error type of thing?

### PR DESCRIPTION
**Pull request by mark-stopka**
Issue
-----------

- This PR is the first contribution towards issue #595

- This PR **does not result** in breaking changes to upstream dependencies. For this PR to be valid **requires** merging of #597.

Checklist
---------
- [ ] This PR contains all the work required to resolve the linked issue.

- [X] The work contained has sufficient documentation to describe what it does and how to do it.

- [ ] The work has sufficient tests and/or testing.

- [X] I have committed clear and descriptive commits. Be considerate as somebody else will have to read these.

- [ ] I have added the appropriate labels to this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ sudo yum install -y curl git openssl-devel systemd-devel zlib-devel # Install re
 **SLES** / **openSUSE**
 
 ```bash
-sudo zypper install -y curl git libopenssl-devel systemd-devel zlib-devel # Install required libraries and their header files required to build software
+sudo zypper install -y curl gcc git gmp-devel libopenssl-devel systemd-devel zlib-devel # Install required libraries and their header files required to build software
```
Comment (erikd) [inline README.md]: Probably best if this commit was quashed down into the previous one, but otherwise looks good. Thanks!

**Comment by erikd** [comment]
Just realised that since this PR has been raised my someone outside the IOHK org, I will been to pull it into the org and raise a new one.

Closing in favor of https://github.com/input-output-hk/cardano-node/pull/599

**Comment by dcoutts** [comment]
What was the restriction exactly? We should fix it.

**Comment by erikd** [comment]
CI des not run on PRs in external (ie non-IOHK) repos.

**Pull request closed without merging**